paystell-frontend/src/components/payment-links/PaymentLinksTable.tsx

```diff
@@ -0,0 +1,160 @@
+                                            : "bg-red-100 text-red-700"
+                                        }`}
+                                >
+                                    {item.state}
+                                </Badge>
+                            </TableCell>
+                            <TableCell className="text-center">
+                                <div className="flex items-center justify-center gap-2">
+                                    <Button variant="ghost" size="icon">
+                                        <Share2 className="h-4 w-4" />
+                                    </Button>
+                                    <DropdownMenu>
+                                        <DropdownMenuTrigger asChild>
+                                            <Button variant="ghost" size="icon">
+                                                <MoreVertical className="h-4 w-4" />
+                                            </Button>
+                                        </DropdownMenuTrigger>
+                                        <DropdownMenuContent>
+                                            <DropdownMenuItem>
+                                                <Edit className="h-4 w-4 mr-2" />
+                                                Edit
+                                            </DropdownMenuItem>
+                                            <DropdownMenuItem>
+                                                <Trash2 className="h-4 w-4 mr-2 text-red-500" />
+                                                Delete
+                                            </DropdownMenuItem>
+                                        </DropdownMenuContent>
+                                    </DropdownMenu>
+                                </div>
+                            </TableCell>
+                        </TableRow>
+                    ))}
+                </TableBody>
+            </Table>
+
+            <div className="flex items-center justify-center gap-2">
+                <Button
+                    variant="ghost"
+                    size="icon"
+                    onClick={() => paginate(currentPage - 1)}
+                    disabled={currentPage === 1}
+                >
+                    <ChevronLeft />
+                </Button>
+                {[...Array(totalSteps)].map((_, index) => (
+                    <Button
+                        key={index}
+                        className={`${currentPage === index + 1
+                                ? "bg-blue-500 text-white" 
+                                : "bg-white border border-blue-500 text-blue-500"
+                            } hover:bg-white hover:text-blue-500 hover:border-blue-500`} 
+                        size="icon"
+                        onClick={() => paginate(index + 1)}
+                    >
+                        {index + 1}
+                    </Button>
```

Comment:
use variant for buttons
```suggestion
                    <Button
                        key={index}
                        variant={currentPage === index + 1
                                ? "default" 
                                : "outline"
                        } 
                        size="icon"
                        onClick={() => paginate(index + 1)}
                    >
                        {index + 1}
                    </Button>
```